common/JailUtil.cpp

```diff
@@ -156,7 +156,7 @@ void cleanupJails(const std::string& root)
         return;
     }
 
-    if (FileUtil::Stat(root + '/' + LO_JAIL_SUBPATH).exists())
+    if (FileUtil::Stat(JailUtil::getLoPath(root)).exists())
```

Comment:
This one seems marginal - is it all tested ? I worry about our test coverage of this generally.

Reply:
Removed. Although it's a thin wrapper to avoid these hard-to-read concatenations and magic constants, I agree that testing wouldn't be cheap. Removed those changes altogether.